crates/ruff/src/rules/pep8_naming/rules/invalid_class_name.rs

```diff
@@ -0,0 +1,69 @@
+use ruff_macros::{define_violation, derive_message_formats};
+use rustpython_parser::ast::Stmt;
+
+use crate::ast::helpers::identifier_range;
+use crate::registry::Diagnostic;
+use crate::source_code::Locator;
+use crate::violation::Violation;
+
+define_violation!(
+    /// ## What it does
+    ///
+    /// Checks for class names that do not follow the `CapWords`/`CamelCase` convention.
```

Comment:
I think that naming convention is commonly referred to as PascalCase. CapWords begs the question what cap stands for (CapitalWords would be a bit more clear).

Reply:
Good point. PEP8 uses less common terminology (e.g. CapWords rather than CamelCase or PascalCase). I wasn't sure  to stick with their usage, but indeed clearer if we use commonly used naming convention names.